### DISCUSSION TITLE
Private traffic not routed by Tunnel in Site to Site IKEV1

### DISCUSSION THREAD

**Discussion by shahidge4** [strongswan/strongswan#1134] [Q&A]
**/etc/ipsec.conf**
config setup
        # strictcrlpolicy=yes
        # uniqueids = no
        charondebug="4"
        plutodebug=all
        plutostderrlog=/var/log/openswan.log
conn %default
        ikelifetime=86400s
        keylife=28800s
        rekeymargin=3m
        keyingtries=1
        keyexchange=ikev1
        authby=secret

conn zong
        aggressive=no
        #pfs=yes
        left=w.x.y.z                 #strongswan outside address
        leftsubnet=192.168.9.209/32         #network behind strongswan
        leftid=w.x.y.z               #IKEID sent by strongswan
        right=a.b.c.d                 #IOS outside address
        rightsubnet=172.25.140.25/32        #network behind IOS
        rightid=a.b.c.d               #IKEID sent by IOS
        auto=add
        ike=aes256-sha1-modp1024           #P1: modp1024 = DH group 2
        esp=aes256-sha1-modp1024           #P2
        leftauth=psk
        rightauth=psk
        keyexchange=ikev1
**cat /etc/strongswan.conf**
charon {
        load_modular = yes
        install_routes = no
        plugins {
                include strongswan.d/charon/*.conf
        }
}

include strongswan.d/*.conf
**ipsec up zong**
initiating Main Mode IKE_SA zong[1] to a.b.c.d
generating ID_PROT request 0 [ SA V V V V V ]
sending packet: from w.x.y.z[500] to a.b.c.d[500] (252 bytes)
received packet: from a.b.c.d[500] to w.x.y.z[500] (132 bytes)
parsed ID_PROT response 0 [ SA V V ]
received NAT-T (RFC 3947) vendor ID
received FRAGMENTATION vendor ID
selected proposal: IKE:AES_CBC_256/HMAC_SHA1_96/PRF_HMAC_SHA1/MODP_1024
generating ID_PROT request 0 [ KE No NAT-D NAT-D ]
sending packet: from w.x.y.z[500] to a.b.c.d[500] (244 bytes)
received packet: from a.b.c.d[500] to w.x.y.z[500] (304 bytes)
parsed ID_PROT response 0 [ KE No V V V V NAT-D NAT-D ]
received Cisco Unity vendor ID
received XAuth vendor ID
received unknown vendor ID: 25:0e:24:47:72:c5:08:84:b2:9d:18:32:cf:29:56:9a
received unknown vendor ID: 1f:07:f7:0e:aa:65:14:d3:b0:fa:96:54:2a:50:01:00
generating ID_PROT request 0 [ ID HASH N(INITIAL_CONTACT) ]
sending packet: from w.x.y.z[500] to a.b.c.d[500] (108 bytes)
received packet: from a.b.c.d[500] to w.x.y.z[500] (92 bytes)
parsed ID_PROT response 0 [ ID HASH V ]
received DPD vendor ID
IKE_SA zong[1] established between w.x.y.z[w.x.y.z]...a.b.c.d[a.b.c.d]
scheduling reauthentication in 86141s
maximum IKE_SA lifetime 86321s
generating QUICK_MODE request 825731759 [ HASH SA No KE ID ID ]
sending packet: from w.x.y.z[500] to a.b.c.d[500] (316 bytes)
received packet: from a.b.c.d[500] to w.x.y.z[500] (92 bytes)
parsed INFORMATIONAL_V1 request 1965302834 [ HASH N((24576)) ]
received (24576) notify
received packet: from a.b.c.d[500] to w.x.y.z[500] (316 bytes)
parsed QUICK_MODE response 825731759 [ HASH SA No KE ID ID N((24576)) ]
selected proposal: ESP:AES_CBC_256/HMAC_SHA1_96/MODP_1024/NO_EXT_SEQ
CHILD_SA zong{1} established with SPIs c3cd48ec_i a1f171c8_o and TS 192.168.9.209/32 === 172.25.140.25/32
**tcpdump -i enp0s31f6 host a.b.c.d**
tcpdump: verbose output suppressed, use -v or -vv for full protocol decode
listening on enp0s31f6, link-type EN10MB (Ethernet), capture size 262144 bytes
03:53:36.477412 IP a.b.c.d.isakmp > w.x.y.z.isakmp: isakmp: phase 2/others ? inf[E]
03:53:36.478240 IP w.x.y.z.isakmp > a.b.c.d.isakmp: isakmp: phase 2/others ? inf[E]
03:53:46.471544 IP a.b.c.d.isakmp > w.x.y.z.isakmp: isakmp: phase 2/others ? inf[E]
03:53:46.472370 IP w.x.y.z.isakmp > a.b.c.d.isakmp: isakmp: phase 2/others ? inf[E]
03:53:56.471267 IP a.b.c.d.isakmp > w.x.y.z.isakmp: isakmp: phase 2/others ? inf[E]
03:53:56.472044 IP w.x.y.z.isakmp > a.b.c.d.isakmp: isakmp: phase 2/others ? inf[E]

Kindly help where i am do wrong

**Comment by strongX509** [comment]
The log entry
````
 CHILD_SA zong{1} established with SPIs c3cd48ec_i a1f171c8_o and TS 192.168.9.209/32 === 172.25.140.25/32
````
shows that the IPsec SA has been successfully established but **tcpdump** just shows periodic DPD (Dead Peer Dectection) IKE packets initiated by the remote host **a.b.c.d** but not any encrypted ESP packets. Did you actually send any plaintext packets (e.g. a **ping**) from **192.168.9.209** to **172.25.140.25** ?
